qiskit_experiments/library/calibration/analysis/fine_half_angle_analysis.py

```diff
@@ -0,0 +1,30 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Fine half angle calibration analysis."""
+
+from qiskit_experiments.curve_analysis import ErrorAmplificationAnalysis
+
+
+class FineHalfAngleAnalysis(ErrorAmplificationAnalysis):
+    r"""Analysis class for the HalfAngle experiment to define the fixed parameters.
+
+    # section: note
+
+        The following parameters are fixed.
```

Comment:
It read funny to me that the parameters are fixed (at first I was thinking fixed to values in the class definition) and then the next line says the parameter is set by the user, so I think it might be better to clarify the parameter can be set by the user but is held fixed during the fit:

```suggestion
        The following parameters are held fixed during fitting.
```